projects/libyaml/libyaml_emitter_fuzzer.c

```diff
@@ -0,0 +1,296 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "yaml.h"
+#include <assert.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef NDEBUG
+#undef NDEBUG
+#endif
+
+#define BUFFER_SIZE 65536
+#define MAX_EVENTS   1024
+
+int compare_events(yaml_event_t *event1, yaml_event_t *event2)
```

Comment:
could this return a bool?
also maybe rename to events_equal ? (compare for me usually signals return value similar to strcmp, so positive/negative/0)